templates/azure/ebook.html

```diff
@@ -0,0 +1,96 @@
+{% extends "azure/base_azure.html" %}
+
+{% block title %}Why do Cloud innovators migrate to Pro?{% endblock %}
+{% block meta_description %}{% endblock %}
+{% block meta_copydoc %}{% endblock meta_copydoc %}
+
+{% block head_extra %}
+  <style>
+    h2,
+    .p-heading--2 {
+      font-weight: 300;
+    }
+  </style>
+{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+  <div class="row">
+    <div class="col-7 u-vertically-center">
+      <div>
+        <h1>Why do Cloud innovators migrate to Ubuntu Pro? {{ utm_campaign }}</h1>
+
+        <p class="u-no-padding--top p-heading--4">Get our e-book to find out</p>
+
+        <p>
+          <a href="#ubuntu-pro-ebook" class="p-button--positive">Download</a>
+        </p>
+      </div>
+    </div>
+    <div class="col-5 u-hide--small u-vertically-center u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/0959ce33-NEW_Ubuntu_ebook_1.png",
+        alt="",
+        width="283",
+        height="401",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+{% include "azure/shared/_azure-logo-strip.html" %}
+
+<div class="p-strip--light">
+  <div class="row">
+    <div class="col-7">
+      <h2 class="p-heading--4">Ubuntu Pro for Azure is a premium image delivering the most comprehensive open source security and compliance</h2>
+
+      <hr class="p-separator u-no-margin--top" />
+
+      <p class="p-heading--3">As the publishers of Ubuntu, we went one step further and developed a premium Ubuntu Pro image in partnership with public cloud providers, to deliver a preferred open-source solution for cloud innovators.</p>
+
+      <p class="p-heading--3">Here is our latest e-book, where we share the details of how Ubuntu Pro on Azure delivers extended open-source security, enterprise compliance, 10 years of extended maintenance and more.</p>
+    </div>
+
+    <div class="col-5" id="ubuntu-pro-ebook">
+      {% with id="4393", returnURL="/azure/thank-you", cta="Get the e-book" %}
+      {% include "engage/shared/_en_engage_form.html" %}
+      {% endwith %}
+    </div>
+  </div>
+</div>
+
+<div class="p-strip">
+  {% include "azure/shared/_how-ubuntu-pro-helps-you.html" %}
+</div>
+
+<div class="p-strip--light">
+  <div class="row">
+    <div class="col-7">
+      <h2>Uplevel Ubuntu journey by activating this private Azure Marketplace offer</h2>
```

Comment:
```suggestion
      <h2>Uplevel your Ubuntu journey by activating this private Azure Marketplace offer</h2>
```

Reply:
?

Reply:
We added a copy doc yesterday, made some suggestions (including this) there.